modern/src/common/components/SelectField.jsx

```diff
@@ -8,9 +8,9 @@ const SelectField = ({
   label,
   fullWidth,
   multiple,
-  value,
-  emptyValue = 0,
-  emptyTitle = '\u00a0',
```

Comment:
Are you sure this works with the regular select? It doesn't have a problem with item height?

Reply:
Currently emptyTitle is not used with the regular `Select` as it was used only in single `Select` and we are using it currently only for `multiple`
`{!multiple && emptyValue !== null && (` <-- so this is now always false for `Select` and that's why I removed it from the regular `Select`. And because of how the `Autocomplete` works, the `emptyTitle` is never actually displayed like this previous code fixed (as you have a clear selection X that 'replaces' this `Select` functionality). The only other way (eg. not an initial empty value) it could display (in list, like `Select` does) is if there is somehow an invalid `option` which is not a correct state anyway (eg. `titleGetter`/`keyGetter` are wrongly configured or one `option` is different Object than the others in the list)